hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouter.java

```diff
@@ -223,10 +223,14 @@ public void testRouterMetricsWhenDisabled() throws Exception {
 
   @Test
   public void testSwitchRouter() throws IOException {
-    assertRouterHeartbeater(true, true);
-    assertRouterHeartbeater(true, false);
-    assertRouterHeartbeater(false, true);
-    assertRouterHeartbeater(false, false);
+    assertRouterHeartbeater(true,true, true);
```

Comment:
Watch the spaces after the comma.

Reply:
In the latest commit e208d71c30e8f1cdc50ba7953bb7f05637a8d2fe , it has been done.

```suggestion
    assertRouterHeartbeater(true, true, true);
```